vanilla-framework/vanilla-framework
# 1484: Add single direction pagination

## Done

- Added support for single direction pagination
- Changed chevron from `background-image` to `content`
- Added new modifiers to `p-pagination__link`: `--previous` and `--next`
- Removed `:first-child` and `:last-child` pseudo-selectors
- Updated pagination example with new classes

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/pagination/
- Remove `p-pagination--next` from the DOM
- Check that the previous button is still styled properly, and that resizing the screen to small will not hide the content
- Refresh
- Remove `p-pagination--previous` from the DOM
- Check that the next button is still styled properly, and that resizing the screen to small makes the next button scale to full-width
- Check that the pattern still matches the [design docs](https://github.com/ubuntudesign/vanilla-design/tree/master/Pagination)

## Details

Fixes #1484
